toshi_hazard_post/aggregation_setup.py

```diff
@@ -34,19 +29,6 @@ def get_vs30s(site_filepath: Union[str, Path]) -> Generator[int, None, None]:
             yield int(site.vs30)  # type:ignore
 
 
-def get_levels(compat_key: str) -> 'npt.NDArray':
```

Comment:
@chrisdicaprio just an observation that we're effectively removing imt levels as part of the compatibility criteria, knowing that interpolation is entirely legit to cope with future variations. I think this level of change record probably  belongs in THS (where the compatibility code resides) so I'll make a start on something we can develop to capture  this knowledge .

Reply:
TBH, I'm not clear on what or if we've decided on how to handle imt levels. Definitely worth a discussion